tests/sorting.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '@playwright/test';
+import { HomePage } from '../pages/home.page';
+import { ProductPage } from '../pages/product.page.ts';
+import { ProductsFiltersFragment } from '../pageFragments/productsFilters.ts';
+import { PowerTools } from '../tools.ts';
+import dotenv from 'dotenv';
+dotenv.config();
+
+
+test.describe('Sorting by name', () => {
+    const sortOptions = [
+      { label: 'Name (A - Z)', ascending: true },
+      { label: 'Name (Z - A)', ascending: false },
+    ];
+  
+    for (const { label, ascending } of sortOptions) {
+      
+      test(`Verify sorting by ${label}`, async ({ page }) => {
+        
+        const homePage = new HomePage(page)
+        const filters = new ProductsFiltersFragment(page);
```

Comment:
ProductsFiltersFragment can not exist without Home page. So you need to initialize ProductsFiltersFragment in the Home page object

```
productsFiltersFragment = new ProductsFiltersFragment(this.page)
```